package.json

```diff
@@ -1,92 +1,94 @@
-		"demo:build": "npm run demo:updateVersion && jekyll build -s demo",
-		"demo:prebuild-version": "cpx 'dist/**/*' demo/release/$npm_package_version/dist --clean && cpx 'doc/**/*' demo/release/$npm_package_version/doc --clean",
-		"demo:prebuild-latest": "cpx 'dist/**/*' demo/release/latest/dist --clean && cpx 'doc/**/*' demo/release/latest/doc --clean",
-		"demo:deploy": "npm run build && npm run jsdoc && npm run demo:prebuild-version && npm run demo:prebuild-latest && npm run demo:build && gh-pages -d demo/_site --add --remote upstream",
-		"demo:updateVersion": "echo $npm_package_version > demo/_data/version.yml",
-		"demo:setup": "cpx 'node_modules/@egjs/common-demo/**/*' 'demo/' && rm demo/package.json",
-		"release": "node config/release.js",
-		"coveralls": "cat ./coverage/lcov.info | ./node_modules/coveralls/bin/coveralls.js",
-		"prepush": "npm run lint",
-		"commitmsg": "node config/validate-commit-msg.js",
-		"changelog": "node config/changelog.js"
-	},
-	"author": {
-		"name": "NAVER Corp."
-	},
-	"namespace": {
-		"eg": "eg"
-	},
-	"license": "MIT",
-	"bugs": {
-		"url": "https://github.com/naver/egjs-view360/issues"
-	},
-	"dependencies": {
-		"@egjs/agent": "^2.1.1",
-		"@egjs/axes": "^2.2.0",
-		"@egjs/component": "^2.0.0",
-		"es6-promise": "^4.1.1",
-		"webvr-polyfill": "^0.9.16"
-	},
-	"devDependencies": {
-		"@egjs/common-demo": "github:naver/egjs#common-demo",
-		"babel-core": "^6.25.0",
-		"babel-eslint": "^7.2.3",
-		"babel-loader": "^7.1.0",
-		"babel-plugin-add-module-exports": "^0.2.1",
-		"babel-plugin-es6-promise": "^1.1.1",
-		"babel-plugin-transform-es3-member-expression-literals": "^6.22.0",
-		"babel-plugin-transform-es3-property-literals": "^6.22.0",
-		"babel-plugin-transform-object-assign": "^6.22.0",
-		"babel-preset-es2015": "^6.24.1",
-		"chai": "^4.0.2",
-		"clean-webpack-plugin": "^0.1.16",
-		"coveralls": "^2.13.1",
-		"cpx": "^1.5.0",
-		"egjs-jsdoc-template": "1.4.0",
-		"eslint": "^3.19.0",
-		"eslint-config-naver": "^1.0.0",
-		"eslint-loader": "^1.8.0",
-		"gh-pages": "^1.0.0",
-		"hammer-simulator": "0.0.1",
-		"husky": "^0.14.1",
-		"inject-loader": "^3.0.0",
-		"istanbul-instrumenter-loader": "^2.0.0",
-		"jsdoc": "^3.4.3",
-		"karma": "^1.7.0",
-		"karma-chai": "^0.1.0",
-		"karma-chrome-launcher": "^2.2.0",
-		"karma-coverage-istanbul-reporter": "^1.3.0",
-		"karma-mocha": "^1.3.0",
-		"karma-mocha-reporter": "^2.2.3",
-		"karma-sinon": "^1.0.5",
-		"karma-sourcemap-loader": "^0.3.7",
-		"karma-webpack": "^2.0.3",
-		"lite-fixture": "^1.0.2",
-		"mocha": "^3.4.2",
-		"sinon": "^2.3.5",
-		"string-replace-webpack-plugin": "0.1.3",
-		"sync-exec": "^0.6.2",
-		"uglify-js": "^2.8.28",
-		"uglifyjs-webpack-plugin": "^0.4.6",
-		"webpack": "^2.6.1",
-		"webpack-dev-server": "^2.5.0",
-		"webpack-merge": "^4.1.0",
-		"write-file-webpack-plugin": "^4.1.0",
-		"xml2js": "^0.4.19"
-	}
+  "name": "@egjs/view360",
```

Comment:
Meaningless changes for indentation should be restricted. It makes difficult to read changes.

Reply:
Okay. I'm going to revert it.